lang/en/entities/inventories.php

```diff
@@ -22,7 +24,7 @@
     ],
     'helpers'       => [
         'copy_entity_entry' => 'Display the item\'s entry instead of the custom description.',
```

Comment:
Not related to the ticket, but we renamed the module to Objects last year so it should be "object\'s entry instead"